exporter.go

```diff
@@ -102,6 +127,29 @@ func (e *Exporter) Export(
 			}
 			bpMD.Layers[layer.name()] = lmd
 		}
+		meta.Buildpacks = append(meta.Buildpacks, bpMD)
+
+		// Process cached layers.
+		bpCacheMD := metadata.BuildpackLayersMetadata{
+			ID:      bp.ID,
+			Version: bp.Version,
+			Layers:  map[string]metadata.BuildpackLayerMetadata{},
+		}
+		for _, layer := range bpDir.findLayers(cached) {
+			if !layer.hasLocalContents() {
+				return fmt.Errorf("failed to cache layer %q because it has no contents", layer.Identifier())
```

Comment:
Is this really an error?

Reply:
This it taken verbatim from the current cacher implementation. I assume they look at it as an error because you shouldn't even create the cache layer if it's empty. This is similar to how exporting launch layers fails when the layer is also cache=true and has no contents.